code/modules/martial_arts/combos/bearserk/smokey.dm

```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/bearserk/smokey
+	name = "Smokey"
+	steps = list(MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_DISARM)
+	explaination_text = "Mentally channel the occultic fury of Smoh'Kie to set your opponent aflame!"
+
+/datum/martial_combo/bearserk/smokey/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(!isliving(target))
+		return
+	user.do_attack_animation(target, ATTACK_EFFECT_PUNCH)
+	target.visible_message("<span class='warning'>[user] sets [target] on fire with otherwordly powers!</span>", \
+						"<span class='userdanger'>As [user] punches you with a searing fist, these words echo in your mind; \"remember... only YOU can prevent forest fires!\"</span>")
+	target.apply_damage(10, BURN, user.zone_selected)
+	playsound(get_turf(user), 'sound/weapons/punch1.ogg', 20, 1, -1)
```

Comment:
Can you include these as keyword arguments and set vary to TRUE instead of 1 for all of these new playsounds?
```suggestion
	playsound(get_turf(user), 'sound/weapons/punch1.ogg', 20, vary = TRUE, extrarange = -1)
```